app/Models/Message.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Gladiator\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Message extends Model
+{
+    /**
+     * Array of available message types.
+     *
+     * @var array
+     */
+    protected static $types = [
```

Comment:
Yeah true, so I'll do generic types and the body copy can always just differ but there are defined general _types_. Good call @angaither